helm_deploy/templates/metabase/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.metabase.enabled }}
+{{- $fullName := include "laa-crime-application-store.fullname" . -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}-metabase
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $fullName }}-metabase
+  template:
+    metadata:
+      labels:
+        app: {{ $fullName }}-metabase
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ .Values.service_account.name }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+      - name: metabase
+        image: metabase/metabase:v0.49.3
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        ports:
+          - containerPort: 3000
+        resources:
+          requests:
+            cpu: 25m
+            memory: 1Gi
+          limits:
+            cpu: 500m
+            memory: 3Gi
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 1000
+          runAsGroup: 1000
```

Comment:
```
securityContext:
  allowPrivilegeEscalation: false
  runAsUser: 1000
  runAsGroup: 1000
```

looking at other config do we need to add this? but not sure what impact it may have
```
  runAsNonRoot: true
  seccompProfile:
    type: RuntimeDefault
  capabilities:
    drop: [ "ALL" ]
```

Reply:
https://dsdmoj.atlassian.net/browse/CRM457-1547